ViewModeSwitch/index.html

```diff
@@ -9,8 +9,7 @@
 		<meta name="keywords" content="view mode, switch, css, style, grid, list, template" />
 		<meta name="author" content="Codrops" />
 		<link rel="shortcut icon" href="../favicon.ico">
-		<link rel="stylesheet" type="text/css" href="css/default.css" />
-		<link rel="stylesheet" type="text/css" href="css/component.css" />
+		<link rel="stylesheet" type="text/css" href="css/css/styles.css" />
```

Comment:
Out of curiosity...css/css/styles.css? Not just css/styles.css? :) The latter will be a little easier for you to organize down the road.